src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -144,11 +151,6 @@
     private final Object participantLock = new Object();
 
     /**
-     * Takes care of conference recording.
-     */
-    private JitsiMeetRecording recording;
```

Comment:
Why was this removed?

Reply:
It was marked as deprecated (for a long time, if I remember it correctly), so I didn't bother migrating it.

Reply:
Ah, that's jirecon-specific. OK, thanks.